src/implementations/twig/components/page-header/eu-page-header-core.story.js

```diff
@@ -0,0 +1,182 @@
+import { withNotes } from '@ecl/storybook-addon-notes';
+import withCode from '@ecl/storybook-addon-code';
+import { correctSvgPath } from '@ecl/story-utils';
+
```

Comment:
same remark as on the footer: there is no EU page header harmonised?
This is strange as there is a site header...